django/db/models/expressions.py

```diff
@@ -153,7 +153,7 @@ class BaseExpression:
     # aggregate specific fields
     is_summary = False
     _output_field_resolved_to_none = False
-    empty_aggregate_value = NotImplemented
+    empty_result_set_value = NotImplemented
```

Comment:
We should move it out of the "aggregate specific fields"  section. Maybe:
```suggestion
    # Func() specific fields.
    empty_result_set_value = NotImplemented
```

Reply:
Right, so let's move it above the `# aggregate specific fields`.